CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -40,6 +40,7 @@ public struct MyCoinView: View {
         })
         .task {
             await viewModel.fetchCoinDetail(id: myCoinState.id)
+            await viewModel.fetchChartPricesData(id: myCoinState.id)
```

Comment:
Please help to re-arrange so these method calls can run in parallel. You can refer to what we've done for the `Home` [here](https://github.com/nimblehq/nimble-crypto-ios/blob/a1b4ed874a0d8900d89108df54bb4e0dd068fe18/CryptoPrices/Sources/Home/Sources/Home/HomeView/HomeViewModel.swift#L59)